packages/0x.js/src/artifacts/DummyToken.json

```diff
@@ -0,0 +1,23 @@
+{
+    "contract_name": "DummyToken",
+    "abi":
+    [
+        {
+            "constant": false,
+            "inputs": [
+                {
+                    "name": "_target",
+                    "type": "address"
+                },
+                {
+                    "name": "_value",
+                    "type": "uint256"
+                }
+            ],
+            "name": "setBalance",
+            "outputs": [],
+            "payable": false,
+            "type": "function"
+        }
+    ]
+}
```

Comment:
I'm pretty sure the DummyToken artifact is bigger then this no? Are we assuming the user you generate a `Token` wrapper instance to access the other methods on a `DummyToken`?

Reply:
Yes

Reply:
ok